Formula/chafa.rb

```diff
@@ -20,8 +21,12 @@ class Chafa < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "freetype"
   depends_on "glib"
   depends_on "imagemagick"
+  depends_on "jpeg-turbo"
+  depends_on "libtiff"
+  depends_on "webp"
```

Comment:
`jpeg`, `libtiff`, `webp` were originally indirectly linked via `imagemagick`, but it looks like upstream is planning to drop `imagemagick` and instead use direct codecs
```suggestion
  depends_on "jpeg-turbo"
  depends_on "libtiff"
  depends_on "webp"
```

From https://github.com/hpjansson/chafa/releases/tag/1.12.0
> The ImageMagick dependency is now optional and deprecated. Packagers are encouraged to drop the ImageMagick dependency (`--without-imagemagick`) and add direct dependencies on relevant image codecs. ImageMagick support will be removed in a future release.